client/src/lib/format.tsx

```diff
@@ -11,6 +11,24 @@ export const formatCurrency = (
   }).format(value);
 };
 
+/**
+ * Parses a currency value into its symbol and amount components
+ * @param value - The numeric value to parse
+ * @param options - Optional Intl.NumberFormatOptions for currency formatting
+ * @returns A tuple containing [currencySymbol, amount] as strings
+ * @example
+ * parseCurrency(1234.56) // returns ['$', '1,234']
+ */
+export const parseCurrency = (
```

Comment:
Do you think we can replace this with `renderCurrency`?

Reply:
Yes that works perfectly 👍